src/Sarif/Baseline/ResultMatching/DataStructures/MatchedResults.cs

```diff
@@ -14,14 +14,20 @@ public class MatchedResults
     {
         private const string MatchResultMetadata_RunKeyName = "Run";
         private const string MatchResultMetadata_FoundDateName = "FoundDate";
+        private const string MatchResultMetadata_PreviousGuid = "PreviousGuid";
 
         public ExtractedResult PreviousResult { get; set; }
```

Comment:
>set [](start = 53, length = 3)

Since you added a constructor that sets PreviousResult and CurrentResult, should the setters be removed? It seems like an odd pattern that this class can basically be 'reused' by re-setting PreviousResult and CurrentResult and then calling one of the Construct() methods. Or is the intent that the instance be resued?

Reply:
+1

---
In reply to: [295535056](https://github.com/microsoft/sarif-sdk/pull/1539#discussion_r295535056) [](ancestors = 295535056)

Reply:
Yeah, that's a good idea. Fixed.

Reply:
Actually, if you do the assignment in the ctor, you don't even need the `private set`. I'll take care of that in my "nit fix" PR.

---
In reply to: [295888531](https://github.com/microsoft/sarif-sdk/pull/1539#discussion_r295888531) [](ancestors = 295888531)